packages/turbo-codemod/src/cli.ts

```diff
@@ -60,14 +60,11 @@ codemodCli
   .parseAsync()
   .then(notifyUpdate)
   .catch(async (reason) => {
-    console.log();
-    if (reason.command) {
-      console.log(`  ${chalk.cyan(reason.command)} has failed.`);
-    } else {
```

Comment:
This was always false, this was a remnant of a way we used to handle errors in create-turbo that got copied across. But we've since stopped with these types of errors.